app/models/product.rb

```diff
@@ -124,7 +124,7 @@ def to_param
   end
 
   def to_s
-    name.present? ? name.html_safe : ''
+    name.presence || ""
```

Comment:
The `#html_safe` was introduced here 273e94102ea3b40ae38bb5b838c526667454c383 to address double-escaping in a receipt view that doesn't seem to exist anymore.